docs/03-running_pvactools.md

```diff
@@ -311,7 +335,147 @@ all \
 /pVACtools_outputs/pvacfuse_predictions \
 --iedb-install-directory /opt/iedb \
 --allele-specific-binding-thresholds \
---percentile-threshold 0.01 \
+--percentile-threshold 2 \
+--run-reference-proteome-similarity \
+--peptide-fasta /HCC1395_inputs/Homo_sapiens.GRCh38.pep.all.fa.gz \
+--problematic-amino-acids C \
+--downstream-sequence-length 100 \
+--n-threads 8 \
+--keep-tmp-files
+```
+
+## Running pVACsplice
+
+pVACsplice is run in order to predict neoantigens from tumor-specific alternative
+splicing patterns. The pipeline uses splice site variants predicted by RegTools
+for this purpose. The RegTools output is used by pVACsplice in combination with a
+GTF file to construct peptide sequences for the alternative splicing patterns and
+extract neoantigens around the splice site.
+
+The pVACsplice pipeline is run using the `pvacsplice run` command.
+
+### Required Parameters for pVACsplice
+
+The `pvacsplice run` command takes a number of required parameters in the
+following order:
+
+- `input_file`: A RegTools junctions output TSV file.
+- `sample_name`: The name of the tumor sample being processed.
+- `allele(s)`: The name of the HLA allele to use for epitope prediction. Multiple
+  alleles can be specified using a comma-separated list. These should be the
+  HLA alleles of your patient. You might have clinical typing information for
+  your patient. If not, you will need to computational predict the patient's
+  HLA type using software such as OptiType.
+- `prediction_algorithms`: The epitope prediction algorithms to use. Multiple
+  prediction algorithms can be specified, separated by spaces. Use `all` to
+  run all available prediction algorithms.
+- `output_dir`: The directory for writing all result files.
+- `annotated_vcf`: A VEP-annotated single- or multi-sample VCF containing
+  genotype and transcript information. This is generally the same input VCF
+  used for pVACseq.
+- `ref_fasta`: A reference DNA FASTA file
+- `gtf_file`: A reference GTF file.
+
+### Optional Parameters for pVACsplice
+
+In addition to the required parameters, the `pvacsplice run` command also offers
+optional arguments to fine-tune your run. You will find a lot of overlap
+between pVACsplice, pVACfuse and pVACseq parameters and the same general considerations
```

Comment:
```suggestion
between pVACsplice, pVACfuse, and pVACseq parameters, and the same general considerations
```
The first comma in this suggestion is just because I like Oxford commas. 🙂   The second is more important to the sentence.